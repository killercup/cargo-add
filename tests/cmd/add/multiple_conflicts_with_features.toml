bin.name = "cargo-add"
args = ["add", "my-package1", "your-face", "--features", "nose"]
status.code = 1
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
