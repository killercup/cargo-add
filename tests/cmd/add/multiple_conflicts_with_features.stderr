Command failed due to unhandled error: Cannot specify multiple crates with features

